### PR TITLE
Added additional step

### PR DESCRIPTION
**Pull request by get-itips**
@scanum

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/deploy/deploy-high-availability-and-disaster-recovery/front-end-pools-for-disaster-recovery.md [hunk 1]
```diff
@@ -34,25 +34,27 @@ You can easily deploy the disaster recovery topology of paired Front End pools u
...
     
    ```
-   <system drive>\Program Files\Skype for Business Server 2015\Deployment\Bootstrapper.exe 
+   <system drive>\Program Files\Skype for Business Server 2019\Deployment\Bootstrapper.exe 
    ```
 
     This configures other services required for backup pairing to work correctly.
     
-9. From a Skype for Business Server Management Shell command prompt, run the following: 
+9. Once Bootstrapper finishes installing required components for backup pairing on every Front end Server in both pools, please be sure to re-apply any existing Cumulative Update that was previously applied on these Front End Servers in both pools and then continue with next step.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/deploy-high-availability-and-disaster-recovery/front-end-pools-for-disaster-recovery.md]:
```suggestion
9. Once Bootstrapper finishes installing the required components for backup pairing on every Front end Server in both pools, please be sure to re-apply any existing Cumulative Update that was previously applied on these Front End Servers in both pools and then continue with the next step.
```

File: Skype/SfbServer/deploy/deploy-high-availability-and-disaster-recovery/front-end-pools-for-disaster-recovery.md [hunk 2]
```diff
@@ -34,25 +34,27 @@ You can easily deploy the disaster recovery topology of paired Front End pools u
...
+
+10. From a Skype for Business Server Management Shell command prompt, run the following: 
     
    ```
    Start-CsWindowsService -Name LYNCBACKUP
    ```
 
-10. Force the user and conference data of both pools to be synchronized with each other, with the following cmdlets:
+11. Force the user and conference data of both pools to be synchronized with each other, with the following cmdlets:
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/deploy-high-availability-and-disaster-recovery/front-end-pools-for-disaster-recovery.md]:
```suggestion
11. Force the user and conference data of both pools to be synchronized with each other using the following cmdlets:
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**